lisa/executable.py

```diff
@@ -154,7 +154,7 @@ def _check_exists(self) -> bool:
         else:
             where_command = "where"
         result = self.node.execute(
-            f"{where_command} {self.command}", shell=True, no_info_log=True
+            f"{where_command} {self.command}", shell=True, sudo=True, no_info_log=True
```

Comment:
When a command run with sudo, the path may be different. It's not safe to add sudo here. If it's needed for main scenario, we can check twice. The second time uses sudo, if the command can be found in this step, the command will be executed with sudo always. Please give me a task, I will update it.